tests/CacheCompat/CommonCache.Test.MsalPython/CommonCache.Test.MsalPython.csproj

```diff
@@ -1,3 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
   <Import Project="../CacheTest.props" />
+  <ItemGroup>
+    <None Update="TestMsalPython.py">
```

Comment:
Is this file already checked in?

Reply:
yes, it just wasn't in the project definition and it needs to be copied to the output on builds.